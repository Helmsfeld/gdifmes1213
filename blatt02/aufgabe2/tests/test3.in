243
937
10