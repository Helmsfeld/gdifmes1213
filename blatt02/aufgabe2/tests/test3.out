400
1662
0
